### PR TITLE
Merge main branch into dev-1.29

### PR DESCRIPTION
**Pull request by katcosgrove**
Merging main into dev-1.29 to maintain branch sync

### CODE REVIEW

**Comment by reylejano** [comment]
clean sync
/approve

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:554]: Doesn't look right.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:524]: Doesn't look right.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:620]: Doesn't look right.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:704]: Doesn't look right.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:783]: Doesn't look right.

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks

/hold

Please remove the merge conflicts (squashing them out), and force-push. Or make an equivalent change.

**Comment by sftim** [comment]
/approve cancel

with https://github.com/kubernetes/website/pull/43908#pullrequestreview-1729317170 in mind. OK to re-approve once this has an appropriate LGTM.

**Comment by sftim** [comment]
@katcosgrove could you squash the latest commit into its (merge commit) parent? Let us know if you've a concern around doing this.

Reason: the feature gates file is one we try to be extra nice about as merge conflicts there are a frustration (which you've kinda seen).

**Comment by katcosgrove** [comment]
Closing this and re-doing due to git being haunted and that being faster than unraveling the problem

**Pull request closed without merging**
